src/app/frontend/logs/logs_controller.js

```diff
@@ -0,0 +1,41 @@
+// Copyright 2015 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Controller for the logs view.
+ *
+ * @final
+ */
+export default class LogsController {
+  /**
+   * @param {!backendApi.Logs} podLogs
+   * @param {!Object} logsColorInversionService
+   * @ngInject
+   */
+  constructor(podLogs, logsColorInversionService) {
+    /** @export {Array<string>} Log set. */
+    this.logsSet = podLogs.logs || [];
```

Comment:
You dont need to add `|| []`. If you do, then change backend code to always send a non null array. Other parts of the code already do this. Look for `make` function call.